doc/playbook.rst

```diff
@@ -101,8 +106,10 @@ The AWS CLI can be used to inspect non-tenenat buckets:
 
```

Comment:
Do these instructions need to be updated? I've been trying to get the file timestamps so I can write up an issue, and even with `AWS_PROFILE` and `LSST_DISABLE_BUCKET_VALIDATION` set:

```
$ s3 --profile rubin-prompt-processing ls s3://rubin-summit/

An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: Unknown
$ s3 --profile rubin-prompt-processing ls s3://rubin:rubin-summit/

Parameter validation failed:
Invalid bucket name "rubin:rubin-summit": Bucket name must match the regex "^[a-zA-Z0-9.\-_]{1,255}$" or be an ARN matching the regex "^arn:(aws).*:(s3|s3-object-lambda):[a-z\-0-9]*:[0-9]{12}:accesspoint[/:][a-zA-Z0-9\-.]{1,63}$|^arn:(aws).*:s3-outposts:[a-z\-0-9]+:[0-9]{12}:outpost[/:][a-zA-Z0-9\-]{1,63}[/:]accesspoint[/:][a-zA-Z0-9\-]{1,63}$"
```

Reply:
Ah, found the problem: to look up `rubin-summit`, you need to **not** use the developer profile, but the default LSST one. 😵‍💫

Reply:
`LSST_DISABLE_BUCKET_VALIDATION` is our stack thing and not recognized by AWS CLI. I don't think we know a way to use AWS CLI with a tenant prefix.  I've been using `mc` for it. Hopefully the updates around line 109 in this PR clarifies it; otherwise I should rephrase better.

Reply:
Sorry, my bad for overlooking the `mc` docs.